src/Education.UnitTests/ErrorList/SonarErrorListEventProcessorTests.cs

```diff
@@ -19,95 +19,97 @@
  */
 
 using Microsoft.VisualStudio.Shell.TableControl;
-using Moq;
 using SonarLint.VisualStudio.Core;
 using SonarLint.VisualStudio.Core.Suppressions;
-using SonarLint.VisualStudio.Education.SonarLint.VisualStudio.Education.ErrorList;
+using SonarLint.VisualStudio.Education.ErrorList;
 using SonarLint.VisualStudio.Infrastructure.VS;
 using SonarLint.VisualStudio.TestInfrastructure;
 
-namespace SonarLint.VisualStudio.Education.UnitTests.ErrorList
+namespace SonarLint.VisualStudio.Education.UnitTests.ErrorList;
+
+[TestClass]
+public class SonarErrorListEventProcessorTests
 {
-    [TestClass]
-    public class SonarErrorListEventProcessorTests
+    private readonly TableEntryEventArgs eventArgs = new();
+
+    private IEducation education;
+    private IErrorListHelper errorListHelper;
+    private IFilterableIssue filterableIssue;
+    private ITableEntryHandle handle;
+    private ILogger logger;
+    private SonarErrorListEventProcessor testSubject;
+
+    [TestInitialize]
+    public void TestInitialize()
     {
-        [TestMethod]
-        public void PreprocessNavigateToHelp_NotASonarRule_EventIsNotHandled()
-        {
-            SonarCompositeRuleId ruleId = null;
-            var handle = Mock.Of<ITableEntryHandle>();
-            var errorListHelper = CreateErrorListHelper(isSonarRule: false, ruleId);
+        education = Substitute.For<IEducation>();
+        errorListHelper = Substitute.For<IErrorListHelper>();
+        handle = Substitute.For<ITableEntryHandle>();
+        filterableIssue = Substitute.For<IFilterableIssue>();
+        logger = new TestLogger(true);
```

Comment:
The `logToConsole` parameter is for Debugging reasons.
```suggestion
        logger = new TestLogger();
```